### PR TITLE
Fix site.webmanifest referencing images that don't exist when built

### PR DESCRIPTION
**Pull request by Steve-Tech**
Hi,

I've never used Vite before so it's probably not the best solution, but after building the web interface, the images `android-chrome-192x192.png` and `android-chrome-512x512.png` in `site.webmanifest` never get replaced with working ones, so I've moved them into the public folder so that they would work.

I've also renamed `safari-pinned-tab.svg` to `favicon.svg` and included it as an icon.

Thanks,
Steve

### CODE REVIEW

File: web/site.webmanifest [hunk 1]
```diff
@@ -1,14 +1,14 @@
 {
-  "name": "",
-  "short_name": "",
```
Comment (NickM-27) [inline web/site.webmanifest:3]: Why were these removed?

Comment (Steve-Tech) [reply web/site.webmanifest:3]: I changed their values to `"Frigate"` instead of leaving them blank.

**Review by NickM-27 — APPROVED** [review]
